src/App.js

```diff
@@ -1,18 +1,28 @@
 import './App.css';
 import React, { useCallback, useRef, useState } from 'react';
 import Crossword from 'react-crossword';
-import { parseSolutionSeedPhrase } from './utils';
+import { parseSolutionSeedPhrase, b64ToUtf8 } from './utils';
 import { parseSeedPhrase } from 'near-seed-phrase';
 import * as nearAPI from "near-api-js";
 import { createGridData, loadGuesses } from "react-crossword/dist/es/util";
 import SimpleDark from './loader';
 
-const App = ({ nearConfig, data }) => {
+const App = ({ nearConfig, data, creatorAccount }) => {
   const crossword = useRef();
   const [solvedPuzzle, setSolvedPuzzle] = useState(localStorage.getItem('playerSolvedPuzzle') || null);
   const playerKeyPair = JSON.parse(localStorage.getItem('playerKeyPair'));
   const crosswordSolutionPublicKey = localStorage.getItem('crosswordSolutionPublicKey');
   const [showLoader, setShowLoader] = useState(false);
+  const [needsNewAccount, setNeedsNewAccount] = useState(false);
+  const [claimError, setClaimError] = useState('');
+
+  async function claimAccountType(e) {
+    if (e.target.value === 'create-account') {
+      setNeedsNewAccount(true);
+    } else {
+      setNeedsNewAccount(false);
+    }
```

Comment:
Could be shortened to:
```suggestion
    setNeedsNewAccount(e.target.value === 'create-account');
```